### PR TITLE
Legacy pages

### PR DESCRIPTION
**Pull request by mattandrews**
This adds an endpoint to return the (currently very basic) content for a legacy page.

### CODE REVIEW

File: config/redactor/Standard.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "buttons": ["formatting", "bold", "italic", "unorderedlist", "orderedlist", "link", "image", "video"],
   "plugins": ["source", "fullscreen", "video"],
-  "toolbarFixed": true
+  "toolbarFixed": true,
+  "allowedTags": ["a", "abbr", "address", "area", "article", "aside", "audio", "b", "blockquote", "br", "button", "canvas", "caption", "cite", "code", "col", "colgroup", "dd", "div", "dl", "dt", "em", "fieldset", "figcaption", "figure", "footer", "form", "h1", "h2", "h3", "h4", "h5", "h6", "header", "hgroup", "hr", "i", "iframe", "img", "input", "label", "legend", "li", "map", "menu", "nav", "ol", "optgroup", "option", "p", "picture", "pre", "s", "section", "select", "small", "source", "span", "strong", "sub", "summary", "sup", "table", "tbody", "td", "textarea", "tfoot", "th", "thead", "time", "title", "tr", "u", "ul", "video"]
```
Comment (davidrapson) [inline config/redactor/Standard.json]: How come you needed to add this?

Comment (mattandrews) [reply config/redactor/Standard.json]: Iframes were being stripped out on save (eg. YouTube clips). Although I also had to disable the HTML Purifier for the field in Craft, which might also have caused this – will double check and confirm.

Comment (mattandrews) [reply config/redactor/Standard.json]: ... just confirmed that it's the Purify setting that was breaking iframes. Have removed this config change!

**Comment by mattandrews** [comment]
@davidrapson is this good to go with that change now?

**Comment by davidrapson** [comment]
👍

**Pull request merged**